src/test/java/com/iexec/core/replicate/ReplicateSupplyServiceTests.java

```diff
@@ -78,16 +75,18 @@
     @Mock private TaskService taskService;
     @Mock private TaskUpdateRequestManager taskUpdateRequestManager;
     @Mock private WorkerService workerService;
-    @Mock private SmsService smsService;
     @Mock private Web3jService web3jService;
     @Mock private ContributionTimeoutTaskDetector contributionTimeoutTaskDetector;
```

Comment:
This detector is no more used in `ReplicateSupplyService`.
Could we clean this test class ?

Reply:
Sure, done in https://github.com/iExecBlockchainComputing/iexec-core/pull/490/commits/b3f5a3128f808b0aa595a83ad09594810d96e7de.